src/database/run-seed.ts

```diff
@@ -0,0 +1,44 @@
+import { DataSource } from 'typeorm';
```

Comment:
this run-seed.ts file does not run while the application startup, right? because you don't include it in main module

Reply:
Yes. It should only run when someone manually runs : "npm run seed"